detect/detect.go

```diff
@@ -412,6 +412,33 @@ MatchLoop:
 			}
 		}
 
+		// check entropy
+		entropy := shannonEntropy(finding.Secret)
+
+		finding.Entropy = float32(entropy)
+		if rule.Entropy != 0.0 {
+			if entropy <= rule.Entropy {				
+				logger.Trace().
+					//Here you can add REDACTED finding or finding.line or leave as is (at your discretion)				
+					Float32("entropy", finding.Entropy).
+					Str("rule-id", rule.RuleID).
```

Comment:
This is already included in the rule's context.
```suggestion
					Str("rule-id", rule.RuleID).
```